server/src/routes/index.ts

```diff
@@ -60,11 +35,55 @@ router.route('/favorite', favoriteRouters);
 router.route('/mapPreview', mapPreviewRouter);
 router.route('/health', healthRoutes);
 
-const helloRouter = new Hono();
-helloRouter.get('/', (c) => {
-  return c.text('Hello, world!');
+const testapi = new Hono();
+
+testapi.get('/', async (c) => {
```

Comment:
Clean up or at least move to route file if you need it for debugging. I'm not against the idea of easy endpoints to quickly debug api status